### PR TITLE
hifi lanscape question player

### PR DESCRIPTION
**Pull request by nikitamarysolomanpvt**
## Explanation
Question player landscape hifi
No changes other than updating the start and end alignment/padding  from 28dp to 52dp of all items in Question player.

Similar like https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/61d64361-df8b-4642-8330-998be097b9de/UP-User-Profile-29

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by veena14cs** [comment]
@nikitamarysolomanpvt Mock link is of exploration content or it is of question player.

**Comment by nikitamarysolomanpvt** [comment]
> @nikitamarysolomanpvt Mock link is of exploration content or it is of question player.

@veena14cs There are no mocks available for question player landscape. The link is provided for reference as per which @mschanteltc suggested alignment change for question player.

**Comment by veena14cs** [comment]
> > @nikitamarysolomanpvt Mock link is of exploration content or it is of question player.
> 
> There are no mocks available for question player landscape. The link is provided for reference as per which @mschanteltc suggested alignment change for question player.

Okay, thanks for clarifying.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Review by rt4914 — CHANGES_REQUESTED** [review]
The AndroidManifest file activity is locked to portrait.

File: app/src/main/res/layout-land/question_player_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/question_recycler_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:divider="@android:color/transparent"
+        android:dividerHeight="8dp"
+        android:overScrollMode="never"
+        android:padding="16dp"
+        android:paddingStart="@dimen/padding_24dp"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/question_player_fragment.xml]: You can directly specify dimension over here instead from dimens file.

File: app/src/main/res/layout-land/question_player_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        android:paddingEnd="@dimen/padding_24dp"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/question_player_fragment.xml]: Ditto here and elsewhere.

File: app/src/main/res/layout-land/question_player_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/end_session_header_text_view" />
+
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/question_recycler_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:divider="@android:color/transparent"
+        android:dividerHeight="8dp"
+        android:overScrollMode="never"
+        android:padding="16dp"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/question_player_fragment.xml]: Please remove this padding if you are using padding start and padding end.

**Comment by veena14cs** [comment]
> The AndroidManifest file activity is locked to portrait.

+1

**Review by veena14cs — CHANGES_REQUESTED** [review]
Suggested changes.

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 3 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/question_player_fragment.xml]: Done.

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 2 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/question_player_fragment.xml]: Done.

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/question_player_fragment.xml]: Done.

**Comment by nikitamarysolomanpvt** [comment]
> The AndroidManifest file activity is locked to portrait.

@veena14cs & @rt4914  its done PTAL

**Review by veena14cs — APPROVED** [review]
LGTM.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**